lib/ae_mdw/names.ex

```diff
@@ -50,16 +50,16 @@ defmodule AeMdw.Names do
 
   @spec fetch_names(pagination(), range(), order_by(), query(), cursor() | nil, boolean()) ::
           {:ok, cursor() | nil, [name()], cursor() | nil} | {:error, reason()}
-  def fetch_names(pagination, range, :expiration, query, cursor, expand?) do
+  def fetch_names(pagination, range, order_by, query, cursor, expand?)
+      when order_by in [:expiration, :deactivation] do
```

Comment:
Maybe I'm not getting this right, but right now these names are being sorted by whatever is on the `ActiveNameExpiration` or `InactiveNameExpiration` table, which just sorts them by expiration regardless of when they were revoked (deactivated). Shouldn't we use a different index for when `by=deactivation`?

Reply:
It's the table name that is not so clear but `InactiveNameExpiration` has either an expiration or a revoke. `ActiveNameExpiration` will always have the expiration.